backend/dal/Migrations/Initial/Up/PostUp/02_PIMS_PROPERTY_LOCATION_VW.sql

```diff
@@ -40,7 +40,6 @@ SELECT PROP.PROPERTY_ID
      , PROP.DESCRIPTION   
      , PROP.REGION_CODE
      , PROP.DISTRICT_CODE
-     , PROP.BOUNDARY.STCentroid() AS GEOMETRY
```

Comment:
Curious as to why this was removed from the view?

Reply:
I didn't even notice this view.  You'll also have to manually run it in DEV otherwise it won't be included.

Reply:
That doesn't seem to be the case, I tested that this is added automatically when the migration runs.

Reply:
Oh, this column was duplicated. Doug removed it.